examples/next/pages/ui/components/liveness/with-credential-provider/index.page.tsx

```diff
@@ -0,0 +1,45 @@
+import dynamic from 'next/dynamic';
+import React from 'react';
+
+import { Amplify } from 'aws-amplify';
+import { AwsCredentialIdentityProvider } from '@aws-sdk/types';
+import { fromCognitoIdentityPool } from '@aws-sdk/credential-providers';
+import awsExports from '@environments/liveness/liveness-environment/src/aws-exports';
+
+import LivenessDefault from '../components/LivenessDefault';
+import Layout from '../components/Layout';
+
+Amplify.configure({
+  ...awsExports,
+  API: {
+    endpoints: [
+      {
+        name: 'SampleBackend',
+        endpoint: 'https://s0ctzw9239.execute-api.us-east-1.amazonaws.com/prod',
+        region: 'us-east-1',
+        // endpoint: '/liveness-next-example/api',
+      },
+    ],
+  },
```

Comment:
Do we need this anymore given we are using BYOB and that is configured in the environment itself?

Reply:
Hm yeah probably not, I just copied this over from the previous one so i'm actually not sure how this works haha

Reply:
yep you're correct, it's not necessary but didnt affect the app because it was configured as a separate api